prow/cmd/phaino/main.go

```diff
@@ -66,9 +67,10 @@ func gatherOptions() options {
 	fs.DurationVar(&o.timeout, "timeout", time.Hour, "Maximum duration for each job (0 for unlimited)")
 	fs.DurationVar(&o.totalTimeout, "total-timeout", 0, "Maximum duration for all jobs (0 for unlimited)")
 	fs.DurationVar(&o.grace, "grace", 10*time.Second, "Terminate timed out jobs after this grace period (1s minimum)")
-	fs.StringVar(&o.gopath, "gopath", defaultGOPATH, "The GOPATH that is used in the container. "+
-		"Default is /home/prow/go, need to be changed if the repository depends on GOPATH and it's is set to a different value in the container.")
-
+	fs.StringVar(&o.gopath, "gopath", defaultCodeMountpath, "The path that is used in the container. "+
```

Comment:
Since this is a command line tool and not versioned, I might prefer keep the legacy arg until it's hard to maintain. Did update the PR with better handling of both args though